include/modem/at_host.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Intellinium <giuliano.franchetto@intellinium.com>
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef AT_CMD_H_
+#define AT_CMD_H_
+
+/**
+ * @file at_host.h
+ *
+ * @defgroup at_host AT command interface driver to register custom handler
+ *
+ * @{
+ *
+ * @brief Public APIs for the AT custom command interface driver.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @typedef at_cmd_custom_handler_t
+ *
+ * This type defines a custom handler which will manage the custom commands
+ * received by the modem.
```

Comment:
I think mentioning the modem in the documentation can be confustion, as the modem has nothing to do with the custom command processing.